m5stick-nemo.ino

```diff
@@ -66,12 +74,14 @@
   #define TINY_TEXT 1
   // -=-=- FEATURES -=-=-
   #define ACTIVE_LOW_IR
-  #define M5LED 19
+  #define M5LED   19
   #define ROTATION
   #define USE_EEPROM
-  #define RTC      //TODO: plus2 has a BM8563 RTC but the class isn't the same, needs work.
```

Comment:
You might need to merge my develop branch into yours, it seems like this PR un-does some stuff I commited over the weekend

Reply:
I reviewd all files now and updated all commits that you merged to the project.. I think it is equals to the main brench (that looks ahead the develop brench, with more changes)